R/read_sc_all_care_home.R

```diff
@@ -43,7 +40,17 @@ read_sc_all_care_home <- function(sc_dvprod_connection = phs_db_connection(dsn =
         .data$period
       )
     ) %>%
-    dplyr::collect()
+    dplyr::select(!c("financial_year", "financial_quarter")) %>%
```

Comment:
Similar here as in home_care - We don't actually use `financial_year` or `financial_quarter` for anything so it's pointless reading them in and doing these corrections!

Reply:
We spoke about this and these variables were needed for correcting period where this was in the format `2017` instead of `2017Q4`. This is now simplified to do this without using `financial_year` or `financial_quarter` which can be removed from the read in. I've pushed the commit and will now merge.